README.md

```diff
@@ -131,10 +130,11 @@ var editor = new MediumEditor('.editable', {
 });
 ```
 
-* __buttons__: the set of buttons to display on the toolbar. Default: `['bold', 'italic', 'underline', 'anchor', 'header1', 'header2', 'quote']`
+* __buttons__: the set of buttons to display on the toolbar. Default: `['bold', 'italic', 'underline', 'anchor', 'h1', 'h2', 'quote']`
 * __diffLeft__: value in pixels to be added to the X axis positioning of the toolbar. Default: `0`
 * __diffTop__: value in pixels to be added to the Y axis positioning of the toolbar. Default: `-10`
 * __firstButtonClass__: CSS class added to the first button in the toolbar. Default: `'medium-editor-button-first'`
+* __headerTags__: the types of html elements to create when the <kbd>H1</kbd>, <kbd>H2</kbd>, or <kbd>H3</kbd> toolbar buttons are clicked.  The first element in the array (index 0) indicates which tag name (ie `<h3>`) will be used when the <kbd>H1</kbd> button is clicked.  The second element in the array (index 1) corresponds to the <kbd>H2</kbd> button and the last element (index 2) corresponds to the <kbd>H3</kbd> button. Default: `['h2', 'h3', 'h4']`
```

Comment:
The previous defaults would have mapped to this:
`headerTags: ['h3', 'h4']`
I've updated the behavior to match medium, so now it's:
`headerTags: ['h2', 'h3', 'h4']`

Is that what you are asking about?